packages/utilities/src/hooks/useConst.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+/**
+ * Hook to initialize and return a constant value.
+ * This is similar to setting a private member in a class constructor.
+ *
+ * @param initialValue - Initial value, or function to get the initial value. Similar to `useState`,
+ * only the value/function passed in the first time this is called is respected.
```

Comment:
Can we mark this `@internal`? Would be nice to have this available for ourselves, but not commit to supporting this API for external consumers in case we ever decide on a different approach.

Reply:
I see your point, but the most immediate usage of this would be in examples, so...we either need to abandon this or be okay with supporting it for awhile.